src/CenterSystem.java

```diff
@@ -1,4 +1,4 @@
-import java.beans.BeanInfo;
+import java.beans.*;
```

Comment:
bad practice

Reply:
I didn't change it ..... maybe some modification in our code causes it